packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -254,27 +295,31 @@ export const PhoneNumberInput = forwardRef<
           <Select
             hideLabel
             autoComplete="tel-country-code"
-            invalid={invalid}
+            invalid={invalid || countryCode.invalid}
             disabled={disabled}
             className={classes['country-code']}
+            aria-readonly={readOnly || countryCode.readonly}
             {...countryCode}
             options={options}
             onChange={eachFn<[ChangeEvent<HTMLSelectElement>]>([
               countryCode.onChange,
               handleChange,
             ])}
             ref={applyMultipleRefs(countryCodeRef, countryCode.ref)}
+            renderPrefix={countryCode.renderPrefix}
           />
           <Input
             hideLabel
             autoComplete="tel-national"
-            placeholder="123 456789"
-            pattern="[0-9]+[0-9 ]+"
+            placeholder={subscriberNumber.placeholder}
+            pattern="^(?:[0-9]\s?){0,13}[0-9]$"
```

Comment:
The maximum number of digits is 15, isn't it? So it should be 14 digits with an optional trailing space + 1 final digit.
```suggestion
            pattern="^(?:[0-9]\s?){0,14}[0-9]$"
```

Reply:
Correct, I was (wrongly) assuming that the max number of digits of the format applied both to country code and subscriber number, but it applies only to the subscriber number 👍🏻